File: Practice problem/return middle word/return middle word.cs
// 연습문제 가운데 글자 가져오기 Level 1

public class Solution {
    public string solution(string s) {
        if (s.Length % 2 == 1) { 
            return s.Substring(s.Length / 2, 1); }
        else { 
            return s.Substring((s.Length - 1) / 2, 2); 
        }
    }
}

File: Practice problem/return middle word/return middle word.java
// 연습문제 가운데 글자 가져오기 Level 1

class Solution {
    public String solution(String s) {
        if (s.length() % 2 == 1) { 
            return s.substring(s.length() / 2, (s.length() / 2)+1);
        }
        else { 
            return s.substring((s.length() - 1) / 2, ((s.length() - 1) / 2)+2); 
        }
    }
}
